Kitodo/src/main/java/org/kitodo/production/forms/TemplateForm.java

```diff
@@ -46,7 +49,7 @@
 import org.kitodo.production.workflow.model.Converter;
 
 @Named("TemplateForm")
-@SessionScoped
+@ViewScoped
```

Comment:
Switch to Viewscoped to that changes to the templates are reflected in the process template list.